doc/github-org-owners.md

```diff
@@ -0,0 +1,49 @@
+## GitHub org owners
+
+The following people have the GitHub "owners" permissions on the NixOS organization:
+<!-- Also keep this in sync with the members of @NixOS/org! -->
+- [@infinisil](https://github.com/infinisil)
+- [@lassulus](https://github.com/lassulus)
+- [@tomberek](https://github.com/tomberek)
+- [@winterqt](https://github.com/winterqt)
+- [@zimbatm](https://github.com/zimbatm)
+## How to contact the team
+For any GitHub-related needs, you can reach out to the org owners by either:
+- Pinging [@NixOS/org](https://github.com/orgs/NixOS/teams/org)
+- [Creating an issue in this repository](https://github.com/NixOS/org/issues/new).
+- Messaging in the [Github org owners help desk Matrix room](https://matrix.to/#/#org_owners:nixos.org).
+
+### Authority and processes
+This team's role is to manage and unblock users of the github.com/NixOS GitHub organization. The @NixOS/steering gives them autonomy to handle small day-to-day tasks and expects them to escalate bigger decisions.
+All org owners can individually take care of implementing:
+- Decisions by bodies that have the authority to make GitHub org changes such as:
+  - Arbitrary decisions by the Steering Committee.
+  - Moderation decisions by the moderation team.
+  - Changes from approved RFCs.
+- Low-impact changes, such as:
+  - Adding new org members to allow review requests.
+  - Creating new unprivileged Nixpkgs teams for mention.
+  - Updating repository meta information.
+
+With approval by at least 2 org owners, they can take care of implementing
```

Comment:
Maybe make it clearer if the 2 includes the implementer.

```suggestion
With approval by at least 2 other org owners, they can take care of implementing
```

or

```suggestion
With approval by at least one other org owner, they can take care of implementing
```

Reply:
The latter, I think this should be even more clear:

```suggestion
Org owners need approval from at least one other org owner to take care of implementing
```